metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-icinga",
-  "version": "5.0.1-rc0",
+  "version": "6.0.0",
```

Comment:
please also generate the changelog with `bundle exec rake release:prepare`

Reply:
Was done. Commit and rake was in the wrong order.